LoopKitUI/Views/Settings Editors/TherapySettingsView.swift

```diff
@@ -372,45 +378,131 @@ struct SectionWithTapToEdit<Content, NavigationDestination>: View where Content:
 // MARK: Navigation
 
 private extension TherapySettingsView {
-    func screen(for setting: TherapySetting) -> some View {
+    
+    func screen(for setting: TherapySetting) -> (@escaping () -> Void) -> AnyView {
```

Comment:
For clarity's sake, can you add a parameter name to the return type here?
i.e. `(_ goBack: @escaping () -> Void) -> AnyView`